### PR TITLE
Fixing import

### PR DESCRIPTION
**Pull request by zxenia**
- allows to create FK related instance on the fly, but not add other fields in this instance
- add column names from current templates

still remains open:
- how to deal with Sample - extraction fields
- some wrong behaviour in admin (see inline comments)

### CODE REVIEW

File: fms_core/resources.py [hunk 1]
```diff
@@ -34,10 +44,30 @@ class Meta:
...
+    biospecimen_type = Field(attribute='biospecimen_type', column_name='Biospecimen Type')
+    name = Field(attribute='name', column_name='Sample Name')
+    alias = Field(attribute='alias', column_name='Alias')
+    volume = Field(attribute='volume', column_name='Volume (uL)', widget=DecimalWidget())
+    concentration = Field(attribute='concentration', column_name='Conc. (ng/uL)', widget=DecimalWidget())
+    depleted = Field(attribute='depleted', column_name='Source Depleted')
+    experimental_group = Field(attribute='experimental_group', column_name='Experimental Group')
+    collection_site = Field(attribute='collection_site', column_name='Collection Site')
+    tissue_source = Field(attribute='tissue_source', column_name='Tissue Source')
+    reception_date = Field(attribute='reception_date', column_name='Reception Data', widget=DateWidget())
```
Comment (zxenia) [inline fms_core/resources.py:57]: I keep on getting error on import for this field:
```
NON FIELD SPECIFIC
“” value has an invalid date format. It must be in YYYY-MM-DD format.
```

checked type and format - date and YYYY-MM-DD, don't know what's the problem

File: fms_core/admin.py [hunk 2]
```diff
@@ -73,6 +73,16 @@ class SampleAdmin(AggregatedAdmin):
         ("Additional Information", {"fields": ("experimental_group", "tissue_source", "phenotype", "comment")}),
     )
 
+    # def individual_participant_id(self, obj):
```
Comment (zxenia) [inline fms_core/admin.py:76]: this is according to the docs FK related objects can be added to list_display [docs](https://docs.djangoproject.com/en/3.0/ref/contrib/admin/#list-display)

but it fails

Comment (davidlougheed) [reply fms_core/admin.py:76]: what are you trying to do with this field?

File: fms_core/models.py [hunk 3]
```diff
@@ -33,7 +33,7 @@ def add_error(errors: dict, field: str, error: ValidationError):
     errors[field] = [*errors.get(field, []), error]
 
 
-barcode_name_validator = RegexValidator(re.compile(r"^[a-zA-Z0-9.-_]$"))
+barcode_name_validator = RegexValidator(re.compile(r"^[a-zA-Z0-9.-_]*$"))
```
Comment (davidlougheed) [inline fms_core/models.py:36]: Change * to +

File: fms_core/resources.py (hunk 1 above)
Comment (davidlougheed) [inline fms_core/resources.py:57]: Reception Data -> Reception Date

Comment (zxenia) [reply fms_core/resources.py:57]: that's what they have in their template

Comment (davidlougheed) [reply fms_core/resources.py:57]: oh ok... we should let them know then.
ill see if i can figure out whats causing the error

**Pull request merged**